alacritty_terminal/src/selection.rs

```diff
@@ -566,7 +566,7 @@ mod test {
         assert_eq!(selection.to_span(&term(5, 10)).unwrap(), Span {
             start: Point::new(0, Column(4)),
             end: Point::new(2, Column(4)),
-            is_block: true,
+            is_block: true
```

Comment:
All changes to this file should be reverted.

Reply:
Whoops, looks like my `rustfmt` did that by mistake